blog/views.py

```diff
@@ -137,14 +140,17 @@ def form_valid(self, form):
     if not is_admin_or_root(self.request.user) and instance.author != self.request.user:
       raise PermissionDenied(_("You don't have the access."))
     with transaction.atomic():
+      if not is_admin_or_root(self.request.user) and review_requested():
+        instance.visible = False
       instance.save()
       instance.revisions.create(title=instance.title, text=instance.text, author=self.request.user)
     return redirect(reverse('blog:detail', kwargs=self.kwargs))
 
 
 class BlogAddComment(LoginRequiredMixin, View):
   def post(self, request, pk):
-    if 'text' in request.POST and request.POST['text']:
+    if 'text' in request.POST and request.POST['text'] \
```

Comment:
Can we hide the textarea on the frontend to prohibit AddComment when `review_requested`?

Reply:
Fixed.